build.sh

```diff
@@ -1,33 +1,29 @@
 #!/usr/bin/env bash
 set -e
 
-npm_package_version="${npm_package_version?Script must be run using npm}"
+declare -a php_versions=(
+	7.2
+	7.1
+	7.0
+	5.6
+)
+declare npm_package_version="${npm_package_version?Script must be run using npm}"
+declare dockerfile_dir
+dockerfile_dir="$(dirname "${BASH_SOURCE[0]}")"
 
 docker login
 
 # NOTE: Not building this stack of images concurrently due to a known issue
 # with docker concurrent builds. https://github.com/moby/moby/issues/9656
-
-docker build \
-  -t "visiblevc/wordpress:latest" \
-  -t "visiblevc/wordpress:latest-php7.2" \
-  -t "visiblevc/wordpress:$npm_package_version-php7.2" \
-./php7.2/
-
-docker build \
-  -t "visiblevc/wordpress:latest-php7.1" \
-  -t "visiblevc/wordpress:$npm_package_version-php7.1" \
-./php7.1/
-
-docker build \
-  -t "visiblevc/wordpress:latest-php7.0" \
-  -t "visiblevc/wordpress:$npm_package_version-php7.0" \
-./php7.0/
-
-docker build \
-  -t "visiblevc/wordpress:latest-php5.6" \
-  -t "visiblevc/wordpress:$npm_package_version-php5.6" \
-./php5.6/
+for php_version in "${php_versions[@]}"; do
+	docker build \
+		--build-arg PHP_VERSION="$php_version" \
+        --build-arg VERSION="$npm_package_version" \
```

Comment:
extra space

Reply:
Whoops. shfmt might have added that break. 

Will fix.